F1WM/ApiModel/News/NewsTag.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace F1WM.ApiModel
+{
+    public class NewsTag
+    {
+        public uint Id { get; set; }
+        public string Title { get; set; }
+        public string Icon { get; set; }
```

Comment:
It returns the string as it is in the database. Not sure how important it is, but we could need to build an actual path to the icon.

Reply:
Still not resolved.